examples/k8petstore/k8petstore.sh

```diff
@@ -16,16 +16,32 @@
 
 echo "WRITING KUBE FILES , will overwrite the jsons, then testing pods. is kube clean ready to go?"
 
+
+#Args below can be overriden when calling from cmd line.
+#Just send all the args in order.
 #for dev/test you can use:
 #kubectl=$GOPATH/src/github.com/GoogleCloudPlatform/kubernetes/cluster/kubectl.sh"
 kubectl="kubectl"
 VERSION="r.2.8.19"
 PUBLIC_IP="10.1.4.89" # ip which we use to access the Web server.
-SECONDS=1000          # number of seconds to measure throughput.
 FE="1"                # amount of Web server  
 LG="1"                # amount of load generators
 SLAVE="1"             # amount of redis slaves 
+TEST_SECONDS="1000"   # 0 = Dont run tests, if > 0,  run tests for n seconds.
+NS="k8petstore"       # namespace
+
+echo "Note that, when running, If public IPs fail, you can use localhost:8080/api/v1beta1/proxy/services/frontend/llen?namespace=$NS and modify this script as necessary."
 
+kubectl="${1:-$kubectl}"
```

Comment:
The params are a great start, but the fact they are positional makes it awkward to pass certain flags while leaving the rest at default.  However I'm not sure how to do an argparse style parser in bash.